lib/ifc-loader.ts

```
@@ -206,6 +206,215 @@ export async function extractDoorOperationTypes(file: File): Promise<Map<number,
+
+            const hasProperties = Array.isArray(pset?.HasProperties) ? pset.HasProperties : []
+            const targetDoorIds = new Set<number>()
+            const relatedObjects = Array.isArray(rel?.RelatedObjects) ? rel.RelatedObjects : []
+            for (const obj of relatedObjects) {
+                const objId = obj?.value
+                if (typeof objId !== 'number') continue
+                if (doorSet.has(objId)) {
+                    targetDoorIds.add(objId)
+                } else {
+                    const doorsForType = typeToDoors.get(objId)
+                    if (doorsForType) doorsForType.forEach((id) => targetDoorIds.add(id))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)